filestore/s3.go

```diff
@@ -46,7 +45,7 @@ func NewS3FileStore(
 	s3client *s3.S3,
 	minioClient *minio.Client,
 	bucket string,
-	disableSSLverify bool,
+	httpClient *http.Client,
```

Comment:
Should add docs for the httpClient in the doc comment above explaining what it's for